db/models/app.model.js

```diff
@@ -1,2 +1,14 @@
 const db = require("../connection");
 
+exports.selectTopics = () => {
+  let sqlQuery = "SELECT * FROM topics;";
+  return db
+    .query(sqlQuery).then(({ rows }) => {
+      return rows;
+    })
+    .catch((err) => {
+      if (err) {
+        return Promise.reject({ status: 404, msg: "Data not found" });
+      }
+    })
```

Comment:
You should only have a catch at the end of the chain - which is at the end of your controller. 

I would remove this catch block anyway but I don't think it's doing anything actually. Your wrong paths are not coming to this model (for example /api/toppics is not coming here). If you remove this catch block your tests should all still pass!